coda/coda_mdstore/views.py

```diff
@@ -639,10 +641,17 @@ def bagProxy(request, identifier, filePath):
     handle = getFileHandle(identifier, filePath)
     if handle:
         resp = HttpResponse(
-            FileWrapper(handle),
             content_type=handle.info().getheader('Content-Type')
         )
         resp['Content-Length'] = handle.info().getheader('Content-Length')
+        if hasattr(settings, 'REPROXY') and settings.REPROXY:
```

Comment:
What if we use getattr to do both these checks?
```suggestion
        if getattr(settings, 'REPROXY', False):
```